ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/validator/LedgerStateAccess.scala

```diff
@@ -48,12 +52,15 @@ trait LedgerStateOperations[LogResult] {
   /**
     * Writes a single log entry to the backing store.  The implementation may return Future.failed in case the key
     * (i.e., the log entry ID) already exists.
+    * @return  offset of the latest log entry
     */
   def appendToLog(key: Key, value: Value): Future[LogResult]
 }
 
 /**
-  * Implements non-batching read and write operations on the backing store based on batched implementations.
+  * Provides default implementations for non-batching read and write operations based on batched operations on the
```

Comment:
```suggestion
  * Provides default implementations for batching read and write operations based on batched operations on the
```

Reply:
I'll reword this -- the point is you need to extend `BatchingLedgerStateOperations` if you support batched operations, otherwise you should extend `NonBatchingLedgerStateOperations`.